ngrok/ngrok.test.w

```diff
@@ -26,8 +26,21 @@ api.get("/uri", inflight () => {
 
 let w = new cloud.Website(path: "./public");
 
-let external = new ngrok.Tunnel(api.url, domain: "eladbgithub.ngrok.dev");
+let external = new ngrok.Tunnel(api.url, 
+  domain: "eladbgithub.ngrok.dev",
+  onConnect: inflight (url) => {
+    log("onConnect called: {url}");
+  }
+);
+
+external.onConnect(inflight (url) => {
+  log("another onConnect callback: {url}");
```

Comment:
I wonder if there's a way to validate that this code is called

(snapshotting logs perhaps? Probably too nondeterministic...)

Reply:
mmmm. The whole testing story here is pretty broken :-(